index.html

```diff
@@ -0,0 +1,195 @@
+
+          <div class="pro_div" id="card4">
+            <h3>Profesional Art Printing Data</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="pro_tags">
+              <span>CSS</span>
+              <span>HTML</span>
+              <span>Bootstrap</span>
+              <span>Ruby</span>
+            </div>
+            <button class="button pro_button" type="button">See project</button>
+          </div>
+
+          <div class="pro_div" id="card5">
+            <h3>Profesional Art Printing Data</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="pro_tags">
+              <span>CSS</span>
+              <span>HTML</span>
+              <span>Bootstrap</span>
+              <span>Ruby</span>
+            </div>
+            <button class="button pro_button" type="button">See project</button>
+          </div>
+
+          <div class="pro_div" id="card6">
+            <h3>Profesional Art Printing Data</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <div class="pro-tags">
+              <span>CSS</span>
+              <span>HTML</span>
+              <span>Bootstrap</span>
+              <span>Ruby</span>
+            </div>
+            <button class="button pro_button" type="button">See project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <section id="about_section">
+      <h2 id="ment">About me</h2>
+      <p id="about">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+        contact me.</p>
+
+      <button class="button" id="resume-button" type="button">Get My Resume</button>
+
+    </section>
+    <section id="ability">
+      <div id="separator"></div>
+      <div id="ability-container">
+        <ul id="languages">
+          <li><span class="bullet" id="languages-bullet"></span>Languages</li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+        <ul id="frameworks">
+          <li class="ability-title"><span class="bullet" id="frameworks-bullet"></span>Frameworks</li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <ul id="skills">
+          <li class="ability-title"><span class="bullet" id="skills-bullet"></span>Skills</li>
+          <li>Codekit</li>
+          <li>GitHub</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
```

Comment:
- [ ]  Your about me section should be similar to the one from figma design, make it responsive as well, and make sure that it doesn't affect your mobile version  design 

Your design 
![image](https://user-images.githubusercontent.com/43877689/178112268-787d2dfc-f334-446f-9ac3-8426b5a88429.png)

Figma design 

![image](https://user-images.githubusercontent.com/43877689/178112279-8235a520-2b9a-4ec8-a52c-137604d7459f.png)